deploy/crds/app.kiegroup.org_kogitojobsservices_crd.yaml

```diff
@@ -0,0 +1,350 @@
+                          divisor:
+                            description: Specifies the output format of the exposed
+                              resources, defaults to "1"
+                            type: string
+                          resource:
+                            description: 'Required: resource to select'
+                            type: string
+                        required:
+                        - resource
+                        type: object
+                      secretKeyRef:
+                        description: Selects a key of a secret in the pod's namespace
+                        properties:
+                          key:
+                            description: The key of the secret to select from.  Must
+                              be a valid secret key.
+                            type: string
+                          name:
+                            description: 'Name of the referent. More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#names
+                              TODO: Add other useful fields. apiVersion, kind, uid?'
+                            type: string
+                          optional:
+                            description: Specify whether the Secret or it's key must
+                              be defined
+                            type: boolean
+                        required:
+                        - key
+                        type: object
+                    type: object
+                required:
+                - name
+                type: object
+              type: array
+            image:
+              description: 'Image definition for the service. Example: Domain: quay.io,
+                Namespace: kiegroup, Name: kogito-jobs-service, Tag: latest Defaults
+                to quay.io/kiegroup/kogito-jobs-service:latest On OpenShift an ImageStream
+                will be created in the current namespace pointing to the given image.'
+              properties:
+                domain:
+                  type: string
+                name:
+                  type: string
+                namespace:
+                  type: string
+                tag:
+                  type: string
+              type: object
+            infinispan:
+              description: Has the data used by the service to connect to the Infinispan
+                cluster.
+              properties:
+                authRealm:
+                  description: Name of the Infinispan authentication realm. This sets
+                    the property infinispan.client.hotrod.auth_realm.
+                  type: string
+                credentials:
+                  description: SecretCredentialsType is the data structure for specifying
+                    credentials within a Secret
+                  properties:
+                    passwordKey:
+                      description: PasswordKey is the password for the user.
```

Comment:
Isn't it rather a key pointing to value in Secret holding password value?

Reply:
Yes, indeed.